### PR TITLE
Audit updates

### PR DESCRIPTION
**Pull request by ferricoxide**
Push this week's updates into ash-linux mainline...

### CODE REVIEW

File: ash-linux/el7/STIGbyID/cat2/files/RHEL-07-030740.sh [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
-# Finding ID:	RHEL-07-030531
-# Version:	RHEL-07-030531_rule
+# Finding ID:	RHEL-07-030530
```
[the remark this answers is not in the document]
Comment (ferricoxide) [reply ash-linux/el7/STIGbyID/cat2/files/RHEL-07-030740.sh]: For the shell files, wasn't super concerned with content updates, just name updates, but, yeah, for completeness, probably ought to sync things up.

Also, why are you proposing `RHEL-07-030530` for `RHEL-07-030740.sh`?

File: ash-linux/el7/STIGbyID/cat2/RHEL-07-010180.sls [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
+#    NIST SP 800-53 Revision 4 :: IA-5 (1) (b) 
 #
 #################################################################
 {%- set stig_id = 'RHEL-07-010180' %}
 {%- set helperLoc = 'ash-linux/el7/STIGbyID/cat2/files' %}
-{%- set targFile = '/etc/login.defs' %}
-{%- set searchRoot = 'ENCRYPT_METHOD' %}
+{%- set cfgFile = '/etc/security/pwquality.conf' %}
+{%- set parmName = 'maxrepeat' %}
+{%- set parmValu = '2' %}
```
[the remark this answers is not in the document]
Comment (ferricoxide) [reply ash-linux/el7/STIGbyID/cat2/RHEL-07-010180.sls]: Both are now wrong. Seems to have been update to `3`. Fixing...

File: ash-linux/el7/STIGbyID/cat1/init.sls [hunk 3]
```diff
@@ -21,9 +19,10 @@ include:
   - ash-linux.el7.STIGbyID.cat1.RHEL-07-020240
   - ash-linux.el7.STIGbyID.cat1.RHEL-07-020310
   - ash-linux.el7.STIGbyID.cat1.RHEL-07-021280
+  - ash-linux.el7.STIGbyID.cat1.RHEL-07-021350
   - ash-linux.el7.STIGbyID.cat1.RHEL-07-021910
-  - ash-linux.el7.STIGbyID.cat1.RHEL-07-030010
-  - ash-linux.el7.STIGbyID.cat1.RHEL-07-030810
+  - ash-linux.el7.STIGbyID.cat1.RHEL-07-030000
```
[the remark this answers is not in the document]
Comment (ferricoxide) [reply ash-linux/el7/STIGbyID/cat1/init.sls]: Hmm... I _thought_ I wholly regenerated the `init.sls` files (by massging the output of `ls -1`). Lemme double-check.

[the remark this answers is not in the document]
Comment (eemperor) [reply ash-linux/el7/STIGbyID/cat1/init.sls]: It's the `sh` file here:

https://github.com/plus3it/ash-linux-formula/pull/287/files#diff-22617be246a557fb06aad6eb80a60358

a bit nitpicky...

[the remark this answers is not in the document]
Comment (ferricoxide) [reply ash-linux/el7/STIGbyID/cat1/init.sls]: Ah. Yeah. I made no content changes to the `.../cat1/*` files other than the bare-minimum necessary to resolve Salt-ID collisions caused by needing to renumber  `.../cat2/*` files. I'm not making _functional_ changes to cat1 handlers in this PR, so, I'll worry about fixing headers in a later PR.

**Pull request merged**